erikbenedict/fruit-preserves
# Feature/store user playlist

## Test

- enter search terms and get suggested songs
- add songs to playlist
- see that only 5 songs can be added to the playlist, rotating out the oldest saved song
- see that when you play a song in the playlist it plays music
- refresh page and see that the user playlist is saved and renders
- click on the 'Check out upcoming Concerts!' button and see that the artist name of the song is logged
- click on the 'Clear Playlist' button and see that the user playlist has been cleared, as well as the local storage data
